### DISCUSSION TITLE
Is it possible to create a window with DataGrid with ComboBoxes?

### DISCUSSION THREAD

**Discussion by vldmrklnk** [AvaloniaUI/Avalonia#9649] [Q&A] [answered]
<img width="1272" alt="Untitled" src="https://user-images.githubusercontent.com/39197664/206315824-d8e2576f-99f1-469f-82e5-e32829b83227.png">
I need something like this. I'm trying to do it according this solution #7086.
But still can't see test ComboBoxes in window.

**Comment by maxkatz6** [comment, marked as answer]
Solution explained in that discussion will work.
Double check if you have included datagrid styles and if you have any data in the data grid.

**Reply by vldmrklnk** [reply]
Yeah, thank you a lot! It was an issue with providing DataGrid with data.
